packages/lesswrong/components/tagging/CoreTagsChecklist.tsx

```diff
@@ -29,10 +28,11 @@ const styles = (theme: ThemeType): JssStyles => ({
   }
 }); 
 
-const CoreTagsChecklist = ({onTagSelected, classes, existingTagIds=[] }: {
+const CoreTagsChecklist = ({onTagSelected, classes, existingTagIds=[], shouldDisplayTag = () => true }: {
```

Comment:
`shouldDisplayTag` is never passed in - is that on purpose? I don't quite understand the changes in this file

Reply:
This is only used on Waking Up side rn (though it's plausible to me that other places should also use it the way we use it - namely filter out tags user can't apply from the list)

Reply:
I'm ok with features that we don't use, but might want to. In this case, they're using a frontpage-like topic, which we would also want to use at some point (moving away from our current kinda-weird setup).

Reply:
@Stvad We can keep the change in this file, but can you add a comment in the code explaining why it's here (specifically that Waking Up uses it even though EAF/LW do not)? This will reduce the chance that someone else deletes this seemingly unused code in the future